s3fs/core.py

```diff
@@ -18,10 +17,14 @@
 
 logging.getLogger('boto3').setLevel(logging.WARNING)
 logging.getLogger('botocore').setLevel(logging.WARNING)
-S3_RETRYABLE_ERRORS = (
-    socket.timeout,
-    trans.ReadTimeoutError, trans.IncompleteReadError
-)
+
+try:
+    from boto3.s3.transfer import S3_RETRYABLE_ERRORS
+except ImportError:
+    S3_RETRYABLE_ERRORS = (
+        socket.timeout,
```

Comment:
OK; well I don't trust that boto necessarily did anything good, but I do trust that actual transfer errors when talking with s3 are rare (unless your local connection goes down, in which case retries won't help).

Didn't pinning the version already solve the test failures?